deploy/utils.sh

```diff
@@ -28,9 +27,11 @@ wait_for_it() {
 
 if [ $PLATFORM = "kubernetes" ]; then
     cli_with_timeout="wait_for_it 600 kubectl"
+    cli_with_timeout_job="wait_for_it 200 kubectl"
```

Comment:
i think we need better solution for this by giving the time as a value **but** since i dont want it will be part of this PR(It will require a lot of file changes), please open a ticket and we will handle it better later.